lametro/templates/base.html

```diff
@@ -110,6 +110,20 @@
 
     {% block extra_js %}{% endblock %}
 
+    <script type="text/javascript">
+        $('label.btn').click(function toggleRadio(e) {
+            // Toggle "active" appearance first, so filtering on active works.
+            $(this).button('toggle');
```

Comment:
I've never seen `.button` before. Is that a jquery thing? Is it toggling a class or something?

Reply:
it's [a bootstrap thing](https://getbootstrap.com/docs/3.3/javascript/#buttons)

Reply:
i can add it in a comment!